### DISCUSSION TITLE
Headless is not set to true by default with puppeteer@22.7.1 with NVM on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by kaspernowak** [spatie/browsershot#844] [Bugs]
With a laravel app on an OpenLiteSpeed virtual host on a Ubuntu 22.04 server, installing the latest puppeteer (22.7.1) with nvm* npm causes spatie/laravel-pdf, which uses browsershot under the hood, to crash. Even when setting the correct path to the node directory used by the laravel project due to using nvm.

After too many hours of debugging, I got it to work with by customizing the browsershot instance like this:
```php
Pdf::default()
  ->withBrowsershot(function ($browsershot) {
    $browsershot
      ->setIncludePath('$PATH:'.config('browsershot.include_path'))
      ->addChromiumArguments([
        'headless=shell'
      ]);
  });
```
I found that `headless=true` also works but `headless=new` does not, neither does`->headless()` or `->headlessNew()`. 
Not setting the headless chromium argument results in error.

My understanding is that Browsershot should launch puppeteer with `headless=true` by default, but this does not seem to be the case. 

So it seems like this is a bug, at least with newer versions of puppeteer.

**Comment by RVP04** [comment]
in which file
Almalinux
